### PR TITLE
Add chore git template and clean up markdown files

### PR DESCRIPTION
**Pull request by junan**
## What happened 👀

We have [chore issue template](https://github.com/nimblehq/git-template/blob/main/.github/ISSUE_TEMPLATE/chore_template.md) in our Git template repo. The android team added it in the [Android template](https://github.com/nimblehq/android-templates/blob/develop/.github/ISSUE_TEMPLATE/chore_template.md) too. So we need to add here too.
 
## Insight 📝

The chore issue template file is the same as our Git [chore issue template](https://github.com/nimblehq/git-template/blob/main/.github/ISSUE_TEMPLATE/chore_template.md) file
 
## Proof Of Work 📹

Screen Shot 2022-03-16 at 11 36 26 AM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ labels: "type : bug"
...
- 
+
 ## Steps to reproduce
 
-1. 
-2. 
-3. 
+1.
+2.
+3.
```
Comment (malparty) [inline .github/ISSUE_TEMPLATE/bug_template.md:20]: Should we keep those spaces? They can be handy in the case someone reuses these bullets (faster to start writing stuff). 
Though I should admit that 100% of the time I would just select this sample text and start writing over it ^^

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,12 +2,12 @@ https://github.com/nimblehq/git-template/issues/??
 
 ## What happened 👀
 
-Describe the big picture of your changes here to communicate to the team why we should accept this pull request. 
- 
+Describe the big picture of your changes here to communicate to the team why we should accept this pull request.
+
 ## Insight 📝
 
 Describe in details how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
```
Comment (malparty) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
Describe in detail how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
```

Can we fix that one too? :D

File: {{cookiecutter.app_name}}/.github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,12 +2,12 @@ https://github.com/nimblehq/git-template/issues/??
 
 ## What happened 👀
 
-Describe the big picture of your changes here to communicate to the team why we should accept this pull request. 
- 
+Describe the big picture of your changes here to communicate to the team why we should accept this pull request.
+
 ## Insight 📝
 
 Describe in details how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
```
Comment (malparty) [inline {{cookiecutter.app_name}}/.github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
Describe in detail how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
```

**Comment by malparty** [comment]
LGTM with 2 minor comments ;-) 
/approve

**Review by malparty — APPROVED** [review]

**Review by Lahphim — APPROVED** [review]

**Pull request merged**